app/backend/approaches/chatapproach.py

```diff
@@ -110,7 +110,7 @@ def get_messages_from_history(
 
         total_token_count = 0
         for existing_message in message_builder.messages:
-            total_token_count += message_builder.count_tokens_for_message(dict(existing_message))  # type: ignore
+            total_token_count += message_builder.count_tokens_for_message(existing_message)
```

Comment:
Lol, the original goal of this PR was just to be able to remove `type: ignore`. Success!